project1-sps/script.js

```diff
@@ -1,4 +1,76 @@
+/* What is the input going to be?
+
+User input either scissors paper or stone
+
+// What should the output be?
+user win or lose compared to computer selection
+// How will the computer's option be randomly generated?
+
+// How many different cases are there?
+User input > computer random generate > check if the input matches or win or lose 
+> return message to user win or lose or tie
+*/
+
+//
+
+// initial bankRoll state
+var roundsWon = 0;
+
+// Global Variables Rock, Paper, Scissors
+var stone = 1;
+var paper = 2;
+var scissors = 3;
+
+var generatingComputerChoice = function () {
+  var computerChoiceInt = Math.floor(Math.random() * 3) + 1;
+  return computerChoiceInt;
+};
```

Comment:
nice and concise with good naming. just a little bit more, you can just `return Math.floor(Math.random() * 3) + 1;` instead of saving it to a variable and then returning that variable